Library/Formula/ffmpeg.rb

```diff
@@ -32,6 +32,7 @@ class Ffmpeg < Formula
   option "with-x265", "Enable x265 encoder"
   option "with-libsoxr", "Enable the soxr resample library"
   option "with-webp", "Enable using libwebp to encode WEBP images"
+  option "with-zmq", "Enable using libzmq"
```

Comment:
Can you elaborate on what this does or remove the option (as the text will be be autogenerated)